code/controllers/subsystem/changelog.dm

```diff
@@ -263,6 +276,7 @@ SUBSYSTEM_DEF(changelog)
 		if(config.githuburl)
 			if(alert("This will open PR #[href_list["openPR"]] in your browser. Are you sure?",,"Yes","No")=="No")
 				return
+			// If the github URL in the config has a trailing slash, it doesnt matter here, thankfully github accepts having a double slash: https://github.com/org/repo//pull/1
```

Comment:
Could perhaps consider writing simple helper class for dealing with URLs instead of leaving trash comments like this one ❤️

Reply:
This comment existed forever. I was just moving it. If you want to write a URL helper class, you are more than welcome to

Reply:
Hey man I'm not a maintainer here